test_room_class.py

```diff
@@ -1,87 +1,152 @@
 import unittest
-from room import Room
+from room import Room, Office
 from people import Person, Fellow, Staff
 from allocator import Dojo
 
-
 class TestRoomClass(unittest.TestCase):
     def setUp(self):
         self.test_dojo = Dojo()
-        self.fellow1 = Fellow()
+        self.fellow1 = Fellow("bryan")
         self.is_office = True
+        self.need_room = True
         self.is_fellow = True
 
+        self.room1 = Room()
+        self.office1 = Office("blue_office")
+        #
         """ creating multiple offices"""
-        self.three_rooms = self.test_dojo.create_room(self.is_office, ['Blue', 'Black', 'Yellow'])
-
-        """ creating an office"""
-        self.one_room = self.test_dojo.create_room(self.is_office, ['Spires'])
+        self.test_dojo.create_room(self.is_office, ['Spires', 'Black', 'Yellow'])
 
-        """ Add a person"""
-        one_person = self.test_dojo.create_person("bryant", "neymar", self.is_fellow)
+        """ creating multiple living spaces"""
+        self.test_dojo.create_room(not self.is_office, ['living_spires', 'living_Black', 'living_Yellow'])
 
-        """ test if creation of rooms is successful"""
     def test_if_created(self):
-        self.assertEqual(self.Object, self.room1)
-        self.assertEqual(self.Object, self.office1)
-
-        """ test if created room is added in list"""
-    def test_room_name_creation(self):
-        self.assertIn('Blue', self.three_rooms.room_name, msg="Blue should be in the room names list in dojo")
+        """ test if creation of rooms is successful"""
+        self.assertIsInstance(self.room1, Room)
+        self.assertIsInstance(self.office1, Office)
 
         """check whether room number increases"""
-    def test_create_room_func(self):
-        initial_len = len(self.three_rooms.room_name)
-        initial_office_len = len(self.three_rooms.office_num)
-        initial_living_len = len(self.three_rooms.living_num)
-        self.dojo_create = self.three_rooms.create_room(self.is_office, ['Blue', 'Black', 'Yellow'])
-
+    def test_creation_of_offices(self):
         """ test multiple room creation"""
```

Comment:
This should be a comment not a doc string. Doc strings are used to describe what a class/function does.